src/hwloc/hwloc_base_util.c

```diff
@@ -734,6 +778,25 @@ hwloc_obj_t prte_hwloc_base_get_obj_by_type(hwloc_topology_t topo, hwloc_obj_typ
     }
 
 #if HWLOC_API_VERSION >= 0x20000
+    /* if we are looking for NUMA, then ignore all the
+     * GPU NUMAs */
+    if (HWLOC_OBJ_NUMANODE == target) {
+        unsigned width, w, cnt;
+        hwloc_obj_t obj;
+
+        width = hwloc_get_nbobjs_by_type(topo, target);
+        cnt = 0;
+        for (w=0; w < width; w++) {
+            obj = hwloc_get_obj_by_type(topo, HWLOC_OBJ_NUMANODE, w);
+            if (200 > obj->os_index) {
```

Comment:
Should we change the `200` to `PRTE_HWLOC_NUMA_OSINDEX_CUTOFF` ?

Reply:
I did - in a separate PR, but thanks for catching it here as well.